README.md

```diff
@@ -241,3 +242,15 @@ ib-sriov supports the following [CNI's Capabilities / Runtime Configuration](htt
 
 EOF
 ```
+# SR-IOV Network Operator
+[SR-IOV Network Operator](https://github.com/openshift/sriov-network-operator) can manages the SR-IOV interfaces on the nodes e.g. change the number of VFs on the node, it can be used also to change the link type for the interfaces ETH to IB and vice versa, the [network policy example](https://github.com/openshift/sriov-network-operator/blob/master/deploy/crds/sriovnetwork.openshift.io_v1_sriovnetworknodepolicy_cr.yaml#L38) shows how to use the operator to change the link type for the given PCI Addresses of PFs.
+
+# System Requirements and Topology
+InfiniBand SR-IOV CNI works with kernel 5.5 which supports rdma namespace isolation and read/write GUID of the VFs.
+The network topology for the InfiniBand SR-IOV CNI it is expected to has a node on the same network with OpenSM, the following diagram captures the InfiniBand SR-IOV topology for a two-node k8s cluster with external OpenSM node.
+
+![picture alt](docs/topology.png "Topology")
```

Comment:
Some comments about the drawing.

- Worker nodes dont run kube API , they run  kubelet (or simply omit these services and just stick to kubernetes master/worker node marking)
- Switch symbols should be the same for ETH and IB, possibly with different colors and labels

Reply:
Updated to the workers to kubelet and the symbol